Documentation~/snap-to-grid.md

```diff
@@ -0,0 +1,13 @@
+# Snap to grid
```

Comment:
The content on this page duplicates the information in the Grid and Snap manual page. Do you think it's worth repeating here?

Reply:
Well it was already in the ProBuilder docs so I assumed it should stay. It's also way shorter and in context, which might be nicer for people.

Reply:
I'll leave it in, then